### ISSUE TITLE
long tokens appears as duplicate

### ISSUE THREAD

**Issue by mobiniusbarnali** [splitwise/TokenAutoComplete#395]
I'm facing this issue on
Samsung galaxy S8 and OnePlus 6T devices.
Any idea how to solve this issue?

**Comment by mgod** [comment]
I see you also commented on #383, but we'll leave this one open for further discussion.

I haven't been able to figure out what the keyboard is doing that causes problems yet, unfortunately. This is definitely a problem with some unusual input provided by this specific keyboard or family of keyboards.

**Comment by mobiniusbarnali** [comment]
This issue happens only when I try to add tokens using addObjectSync() method.

And did you found any fix for this issue?

**Comment by mgod** [comment]
No, sorry. I haven't been able to reproduce this myself, so it seems like the next step is to put together some system that would let you record keyboard input through the InputConnection to the view, but I'm not going to have time to put something together soon.
